DigitalLearningSolutions.Web.Tests/Helpers/CustomPromptHelperTests.cs

```diff
@@ -54,5 +54,39 @@ public void GetCustomFieldViewModelsForCentre_returns_populated_list()
                 result[1].Options.Count().Should().Be(0);
             }
         }
+
+        [Test]
+        public void GetCustomFieldViewModelsForCentre_returns_correctly_mapped_answers_with_gap_in_prompts()
+        {
+            // Given
+            var answer1 = "Answer1";
+            var answer2 = "Answer2";
+            var answer3 = "Answer3";

```

Comment:
I think these can be const (IDE warnings)

Whilst we're there, could we also fix the other warnings in this test file? 🙏